README.md

```diff
@@ -63,6 +63,19 @@ you we created a parser that is local. You can open it by copying the path /
 browser then upload the claim.json file of that run.
 ![overview](docs/assets/images/htmlpage.png)
 
+## Upload a previous feedback to the html parser page
+
+A new feature that we did that allow users to upload a previous feedback that they
```

Comment:
Suggestion: a reader that is not aware of the project may find difficulties with trying to guess what's the "new feature you did", probably here it's better to add some links to the feature (the HTML report) and put the correct names.

Reply:
we have explanation about the html parser so i wrote that comments right after them